### PR TITLE
zh translation: index

### PR DESCRIPTION
**Pull request by helight**
zh translation: /index.rst
ref: https://github.com/cloudnativeto/envoy/issues/100

### CODE REVIEW

File: docs/root/index.rst [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
+Envoy 文档
 =================================
 
 .. ifconfig:: release_level in ('pre-release')
 
   .. attention::
 
-    This is pre-release documentation. There is risk of it not being consistent with what is
-    currently implemented in Envoy, though we try to make things consistent as quickly as possible.
+    这是预发布文档。尽管我们尽可能快地在跟进和保持一致，但也有可能没有包含当前 Envoy 新实现的内容。
```
Comment (rootsongjc) [inline docs/root/index.rst]:
```suggestion
    这是预发布文档。这有可能与 Envoy 目前实现的内容不一致，但我们力求尽可能一致。
```

Comment (helight) [reply docs/root/index.rst]: ok，确实更顺一点。

**Pull request merged**